Frends.PDF.Create/Frends.PDF.Create/Definitions/Enums.cs

```diff
@@ -0,0 +1,168 @@
+﻿#pragma warning disable CS1591 // Some of the enums are self-explanatory.

```

Comment:
This should be moved to the exact enum where this is true (in other words right there where PageSizeEnum is). And right after enum you should also have #pragma warning enable CS1591 to restore validations.

Reply:
Set the pragmas for each self-explanatory enums.